### ISSUE TITLE
Turnkey gitlab errors on install

### ISSUE THREAD

**Issue by liamgibbins** [turnkeylinux/tracker#1746]
ones its installed and you ssh in to the configuration it throws errors.

caught an exception and lists lots of lines with line numbers

Traceback (most recent call last):   File               │────────────────────────
                      │ "/usr/lib/inithooks/bin/dialog_wrapper.py", line 75, in  │  
                      │ wrapper     retcode = method("                           │  
                      │ " + text, *args, **kws)   File                           │  
                      │ "/usr/lib/python3/dist-packages/dialog.py", line 2727,   │  
                      │ in inputbox     kwargs, strip_xdialog_newline=True,      │  
                      │ raw_help=True)   File                                    │  
                      │ "/usr/lib/python3/dist-packages/dialog.py", line 1703,   │  
                      │ in _widget_with_string_output     code, output =         │  
                      │ self._perform(args, **kwargs)   File                     │  
                      │ "/usr/lib/python3/dist-packages/dialog.py", line 1504,

there is more than the above.

installed on proxmox

**Comment by JedMeister** [comment]
Unfortunately, our GitLab appliance is currently pretty broken. GitLab is such a finicky beast and is a resource hog (so building/rebuilding is slow and painfull) hence why it has been somewhat neglected for a while. I need to sit down with it and try to work out what is wrong...

Having said that, my guess is that error is not related to GitLab itself, but is actually related to either there being too much text to display in the terminal window you're using, or your terminal session is an unsupported size. Personally, I always just log in via SSH after creating the LXC container on Proxmox and I find that seems to work really well.

Regardless, thanks for taking the time to report the issue.

**Comment by JedMeister** [comment]
A user has recently [posted a potential fix/workaround for the 50x error when GitLab starts](https://www.turnkeylinux.org/comment/51407#comment-51407)! ([source](https://gist.github.com/aursu/fdda2c9e6e26ba7fc2d903ae5f06e4d6)). To save you a click:

```
gitlab-rails c
settings = ApplicationSetting.last
settings.update_column(:runners_registration_token_encrypted, nil)
settings.update_column(:encrypted_ci_jwt_signing_key, nil)
settings.save!
exit
gitlab-ctl restart
```

Please note that this is untested by me, but reports suggest that it works fine.

**Comment by JedMeister** [comment]
Closed by the v17.0/v17.1 release.
